doc/source/schema/schema.md

```diff
@@ -111,6 +156,44 @@ Out of sample posterior predictive samples p(y'|y). Samples should match `poster
 #### `predictions_constant_data`
 Model constants used to get the `predictions` samples. Its variables should have a counterpart in `constant_data`. However, variables in `predictions_constant_data` and their counterpart in `constant_data` can have different coordinate values.
 
+:::{admonition} Note on sample stats, warmup and unconstrained groups
+:class: note, dropdown
+
+The schema does not define which warmup or unconstrained groups exist or can exist
+by default. We recognize both the samplers and the models are continuously evolving.
+Some models already require the use of sampling algorithms to get prior samples,
+in which case we basically need to treat the prior and posterior groups in the same way.
+
+We define the prefixes and to allow libraries that use InferenceData to be aware
+of the potential relations and hopefully support as many cases as possible.
+Back to the case above, it might be necessary to generate a pair plot
+for prior samples generated with NUTS _and_ its associated divergences,
+which would then come from `sample_stats_prior`.
+:::
+
+#### Sample stats groups
+Information and diagnostics for the samples in any InferenceData group
+other than the posterior should be stored in a separate group with the
+`sample_stats_` prefix. For example `sample_stats_prior`.
+
+The same rules and conventions defined in {ref}`schema/sample_stats` apply to
+any sample stats group.
+
+#### Warmup groups
+Samples generated during the adaptation/warmup phases of algorithms like HMC
+can also be stored in InferenData. In such cases, the data/samples
+generated during the adaptation process should be stored in groups with
+the same name with the `warmup_` prefix, e.g. `warmup_posterior`, `warmup_sample_stats_prior`.
```

Comment:
How would the naming go for unconstained warmup groups

    warmup_unconstrained_posterior

?

Reply:
yes, I would make the `warmup_` prefix prepend the other two potential prefixes. I'll add that.